ui.tests/test-module/specs/page/page.runtime.spec.js

```diff
@@ -0,0 +1,32 @@
+/*******************************************************************************
+ * Copyright 2024 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
```

Comment:
Please change the spec to pagev2.runtime.spec.js

Reply:
done